js/index.js

```diff
@@ -92,7 +92,17 @@ async function buildQuickTryUI() {
     xhr.onload = function () {
         if (xhr.readyState === 4 && xhr.status === 200) {
             config = toml.parse(xhr.responseText);
-
+            var example = urlParams.get("example");
```

Comment:
TOML and the rest of the code use the word "app", can we use the same here?
```suggestion
            var requestedApp = urlParams.get("app");
```

Reply:
I renamed it to app, thanks for the review!